src/lib/Scenes/Artist/Artist.tsx

```diff
@@ -122,6 +123,13 @@ export const ArtistQueryRenderer: React.FC<ArtistQueryRendererProps> = ({ artist
               }
               ...ArtistHeader_artist
               ...ArtistArtworks_artist
+              auctionResultsConnection(first: 1) {
+                edges {
+                  node {
+                    internalID
+                  }
+                }
+              }
```

Comment:
I wouldn't hold up a merge for this, but it looks like there is a `totalCount` field on `auctionResultsConnection` that could be used to identify if the auction has results. I'd prefer that because it's a little more direct about what we're querying. (Associated changes would have to be made if you decide to go this route.)
 
```suggestion
              auctionResultsConnection {
                totalCount
              }
```

Reply:
You're absolutely right. I will change it 🙏